### ISSUE TITLE
build_imagenet_data.py Found 0 images with bboxes...

### ISSUE THREAD

**Issue by ahyunSeo** [tensorflow/models#4051]
I'm working on preprocessing ImageNet dataset.
Mainly I'm working on models/research/slim/datasets directory of the repository.

I had my own download so I removed DOWNLOAD part in download_and_convert_imagenet.sh 
even after bazel build the script failed at running python files,
so I ran python scripts **one-by one**
succesfully ran **preprocess_imagenet_validation_data.py** (It only organizes the subdir)
and ran **process_bounding_boxes.py**,
I obtained proper bounding box csv with this message (also this msg are the correct one according to the comment)

**Wrote 615299 bounding boxes from 544546 annotated images.**

But I ran **build_imagenet_data.py** I got 

**Found 0 images with bboxes out of 2500000 images**

at the point of processing train set it detects N images with bbox but
at the point of processing validation set it detects 0 bboxes.
I manually checked csv file and it only contains entry of n000*****_*****.JPEG,
and **because of the basename of my validation set ILSVRC2012_val_00000***, It detects None.**
Is this normal?

I'm starting this issue because I got almost-zero evaluation accuracy running this repo,
https://github.com/mfigurnov/sact

TF ver : v1.7.0-3-g024aecf414 1.7.0
CUDA 9.0 cudnn 7.x w/ NVIDIA Titan XP (12GB) * 2

Summary of the question
1. Does process_bounding_boxes.py supposed to process only training set?
2. Does build_imagenet_data.py supposed to find bboxes for training set only?

**Comment by vilmara** [comment]
@ahyunSeo for the message "Found 0 images with bboxes out of 2500000 images" doublecheck the path to file with bounding boxes annotations, make your the script is able to find and read this file.

Also, as the scripts were written, the imagenet_2012_bounding_boxes.csv file is generated based on the labels found in the file imaganet_lsvrc_2015_synsets.txt, which should contain only labels of the training set, and it should be in this way since the validation set is just a subset of what is already labeled in the training set.

I am starting to run the models in validation mode and getting almost zero accuracy too (example testing resnet50 Accuracy @ 1 = 0.0011 Accuracy @ 5 = 0.0048), please let me know if you get better results.

I hope this help and correct me please if I was wrong or missed some details in the above statement.

Vilmara

**Comment by vilmara** [comment]
@ahyunSeo, have you increased the number of epochs to see if the evaluation accuracy improves?, I am still working on it to get better accuracy

**Comment by ahyunSeo** [comment]
@vilmara No I'm not... I'm pretty sure there's something wrong in my tfslim dataset. Actually I'm considering to reproduce the code to pytorch.

**Comment by ahyunSeo** [comment]
@nealwu  Can you help me out with this problem?

**Comment by tensorflowbutler** [comment]
Hi There,
 We are checking to see if you still need help on this, as this seems to be considerably old issue. Please update this issue with the latest information, code snippet to reproduce your issue and error you are seeing.
 If we don't hear from you in the next 7 days, this issue will be closed automatically. If you don't need help on this issue any more, please consider closing this.
